src/StreamFactoryInterface.php

```diff
@@ -7,17 +7,32 @@
 interface StreamFactoryInterface
```

Comment:
Why do you think so? The implication in PSR-7 is that all streams are backed by resources.